web/packages/teleport/src/Assist/Conversation/AccessRequests/AccessRequests.tsx

```diff
@@ -0,0 +1,181 @@
+/**
+ * Copyright 2023 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import Flex from 'design/Flex';
+import { ButtonPrimary, ButtonSecondary } from 'design';
+
+import { AccessRequestEvent, AccessRequestStatus } from 'teleport/Assist/types';
+import { TimelineItem } from 'teleport/Assist/Conversation/AccessRequests/TimelineItem';
+
+interface AccessRequestsProps {
+  status: AccessRequestStatus;
+  summary: string;
+  username: string;
+  events: AccessRequestEvent[];
+  created: Date;
+}
+
+const ExpandButton = styled.button`
+  border: none;
+  position: absolute;
+  background: none;
+  font-family: ${p => p.theme.font};
+  left: 50%;
+  transform: translateX(-50%);
+  bottom: 10px;
+  cursor: pointer;
+  font-size: 12px;
+  color: ${p => p.theme.colors.text.main};
+  opacity: 0.8;
+  display: flex;
+  align-items: center;
+  gap: 5px;
+  z-index: 3;
+  text-shadow: 0 0 2px black;
+`;
+
+const Container = styled.div`
+  padding: 10px 15px;
+  width: 100%;
+  box-sizing: border-box;
+  max-height: ${p => (p.expanded ? '1000px' : '109px')};
```

Comment:
`109px` is a very specific value, I think it'd be a good idea to document why it's 109px specifically if you want to keep it.